### DISCUSSION TITLE
Sailing Together ⛵  GitHub Actions 🚢  Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#56927] [Actions]
We're thrilled to dive into the vibrant month of June, brimming with excitement and endless possibilities! Before we embark on this thrilling journey, let's take a moment to shine a spotlight on the extraordinary members of our community who have been beacons of support, fostering an environment of collaboration and empowerment. 

## ✨ Meet the Community Champions: Hear the Thunderous Shoutouts! 📢

- A BIG round of applause for [@Yarden-zamir](https://github.com/Yarden-zamir) for Saving the Day for [@GORA-SAG](https://github.com/GORA-SAG) who couldn’t get the [workflow run correctly](https://github.com/orgs/community/discussions/55027). They shared their Actions expertise with extensive use of Docs. 💗 
- Thank You [@zura101](https://github.com/zura101) for coming back to their [discussion](https://github.com/orgs/community/discussions/56345) related to Runner Controllers and sharing the solution with us. What an incredible self-solve 💯
- Sparkles✨ for [@jge162](https://github.com/https://github.com/jge162) for doing a solid diagnosis of the issue helping [@https://github.com/[allwalte](https://github.com/allwalte)](https://github.com/allwalte) use `nix-shell` for custom shell with a [solid answer](https://github.com/orgs/community/discussions/55678).
- [@noaKurman](https://github.com/noaKurman) had [trouble creating](https://github.com/orgs/community/discussions/55449) and referencing a Docker filename, our superstar ⭐ [@ChristopherHX](https://github.com/ChristopherHX) doubled down on the issue at a glance - Thank You! for sharing the syntax. 👏 

### [GitHub Actions Importer – Bamboo Server and Data Center Migrations (Public Beta)](https://github.blog/changelog/2023-05-22-github-actions-importer-bamboo-server-and-data-center-migrations-public-beta/)

Exciting news! The public beta for migrating Bamboo Server and Data Center to GitHub Actions is now live! Seamlessly plan, test, and automate the migration of your valuable Bamboo pipelines using the user-friendly GitHub Actions Importer. Best of all, this powerful feature comes to you free of charge.

For details on how to get started, check out our [documentation](https://gh.io/actions-importer-bamboo).

### [Actions Runner Controller (Public Beta)](https://github.blog/changelog/2023-05-10-github-actions-actions-runner-controller-public-beta/)

Unleash the power of auto scaling self-hosted runners with Actions Runner Controller and [Runner Scale Sets](https://github.com/actions/actions-runner-controller/releases/tag/gha-runner-scale-set-0.4.0) in public beta. Learn more about scaling self hosted runners in this [GitHub Doc](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller).

## 🤝  Help us build your Actions know-how [Join the FAQ](https://github.com/orgs/community/discussions/48283)❓

Are you constantly grappling with questions and occasional challenges while working with Actions? Share your queries and common issues [in this discussion](https://github.com/orgs/community/discussions/48283), and let's collaboratively build an invaluable resource packed with top Actions FAQs. 

Together, we can create an evergreen reference that addresses your concerns and empowers the Actions community. Let's join our hands in shaping a knowledge hub that ensures smooth sailing in your Actions 🚢 endeavors!

## ☝️ GitHub Availability Report ➕

At GitHub, we put developers first and work hard to provide a safe, open, and inclusive platform for code collaboration. We are committed to be as transparent as we can and here to provide you updates on any disruption of services and with that in mind, we publish GitHub availability reports each month owning and explaining what happened and how we mitigated each specific incident. 

On May 9, we had an incident that caused 8 of the 10 services on the status portal to be impacted by a major (status red) outage. We [shared all the details](https://github.blog/2023-05-16-addressing-githubs-recent-availability-issues/) of what happened, why it happened and what we are doing to prevent it in the future.

We update our [status page](https://www.githubstatus.com/) in real time in case of an ongoing incident about Actions. If you see an open incident, this means that we are looking into it already. If not, please let us know about it in the [GitHub community](https://github.com/orgs/community/discussions/categories/actions).

## Automating Microsoft Power Platform deployment using GitHub Actions 🧑‍💻

Discover 🌐  the synergy of [Microsoft Power Platform](https://powerplatform.microsoft.com/) and GitHub in automating Power Platform deployments! Watch [@scubaninja](https://github.com/scubaninja) delve into leveraging the low-code tools of Power Platform and the robust CI/CD capabilities of [GitHub Actions](https://github.com/features/actions) in [this blog post](https://github.blog/2023-05-24-how-to-automate-a-microsoft-power-platform-deployment-using-github-actions/). You’ll get to learn how to efficiently build custom solutions for your organization and streamline code management, security, and delivery.

**Comment by jsoref** [comment]
@Akash1134,
```
Sparkles✨ for [@jge162](https://github.com/https://github.com/jge162) for doing a solid diagnosis of the issue helping [@https://github.com/[allwalte](https://github.com/allwalte)](https://github.com/allwalte) use nix-shell for custom shell with a [solid answer](https://github.com/orgs/community/discussions/55678).
```

... has a stray `@` before `https`

**Reply by jsoref** [reply]
In https://github.com/orgs/community/discussions/57232, `:hands` isn't a proper emoji reference:
> One or both solutions may be suitable, based on your needs. :hands

**Reply by Akash1134** [reply]
Thanks for highlighting this @jsoref I've seen this issue before. While linking in Markdown, it is adding the URL twice. I'll raise it to the team if I notice more examples.
